Automattic-Tracks-iOS/TracksEvent.m

```diff
@@ -206,7 +206,7 @@ - (BOOL)validateDeviceProperties:(id *)ioValue error:(NSError * __autoreleasing
 
 - (BOOL)validateUserProperties:(id *)ioValue error:(NSError * __autoreleasing *)outError
 {
-    if (ioValue == nil || ([(NSDictionary *)*ioValue count] > 0)) {
+    if (ioValue != nil && ([(NSDictionary *)*ioValue count] > 0)) {
```

Comment:
I think you could omit checking `ioValue != nil` in these three conditionals. Anytime `ioValue == nil` the `[ioValue count] > 0` check will return false.